neurom/apps/morph_stats.py

```diff
@@ -112,7 +122,13 @@ def extract_dataframe(neurons, config):
     if 'neuron' in config:
         del config['neuron']
 
-    stats = {nrn.name: extract_stats(nrn, config) for nrn in neurons}
+    func = partial(_run_extract_stats, config=config)
+    if n_workers == 1:
+        stats = dict(map(func, neurons))
+    else:
+        with multiprocessing.Pool(n_workers) as pool:
+            stats = dict(pool.imap_unordered(func, neurons))
+
     columns = list(next(iter(next(iter(stats.values())).values())).keys())
 
     rows = [[name, neurite_type] + list(features.values())
```

Comment:
by the way, I benchmarked this one day and using numpy arrays if you have the same types for all data is MUCH MUCH MUCH more efficient for this constructor : 
```
pd.DataFrame(columns=['name', 'neurite_type'] + columns, data=rows)
```
and you benefit a lot in doing the conversion beforehand. 
 ```
pd.DataFrame(columns=['name', 'neurite_type'] + columns, data=np.asarray(rows))
```

Reply:
see : 
https://github.com/BlueBrain/libsonata/pull/99#issuecomment-656694175

Reply:
I prefer to stick to the idiomatic way of doing it if you don't mind...

Reply:
I don t mind